cypress/e2e/faq/faq.js

```diff
@@ -0,0 +1,84 @@
+describe("Checking information on every category faq", () => {
+  beforeEach(() => {
+    cy.visit("http://localhost:3000/faq");
+  });
+
+  it("gives all information without filter category (last update August 16, 2021)", () => {
+    cy.get("div.space-y-4 > div")
```

Comment:
Please avoid using inaccessible queries like this. We have better queries provided by `@testing-library/cypress`. Please take a look at the other e2e test file for it.

Reply:
So should I remove the test which gives all the information without filter categories? and after I looked at the documentation on @testing-library/cypress I think it's more for integration and unit tests. Am I right?

Reply:
Okay, I have deleted the test